main.css

```diff
@@ -1,76 +1,442 @@
-* {
-  margin: 0;
-  padding: 0;
-}
-
 @media (min-width: 480px) {
```

Comment:
- [ ] This value of `480px` is too small for your min-width. I kindly suggest that your min-width should be between `768 and 1440`.